### PR TITLE
feat(hooks): Add useStatelessReducer

### PR DESCRIPTION
**Pull request by seanhealy**
### Added

- Added `useStatelessReducer` hook, to help manage controlled component values

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/hooks/src/useStatelessReducer/useStatelessReducer.mdx [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+name: useStatelessReducer
+menu: Hooks
+route: /hooks/use-stateless-reducer
+---
+
+import { Playground } from "docz";
+import { useState } from "react";
+import { useStatelessReducer } from "@jobber/hooks";
+import { Form } from "@jobber/components/Form";
```
Comment (eddysims) [inline packages/hooks/src/useStatelessReducer/useStatelessReducer.mdx:10]: `Form` isn't used in this mdx. Can be removed

**Review by eddysims — CHANGES_REQUESTED** [review]
@seanhealy one very minimal thing here. Other then that looks good!

File: packages/hooks/src/useStatelessReducer/useStatelessReducer.mdx (hunk 1 above)
Comment (joelwarrington-jobber) [reply packages/hooks/src/useStatelessReducer/useStatelessReducer.mdx]: resolved in fc41fb0

**Comment by eddysims** [comment]
@seanhealy doing some cleanup. Going to close this as it hasn't been needed in almost 1 year. If you want some `Hacktoberfest 2021` swag, feel free to redo this.

**Pull request closed without merging**